content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -165,16 +195,33 @@ To help protect a validator from [DDoS](https://en.wikipedia.org/wiki/Denial-of-
 7. Use the [Peer Crawler](peer-protocol.html#peer-crawler) endpoint on one of your stock `rippled` servers. The response should not include your validator. This verifies that your validator's `[peer_private]` configuration is working. One of the effects of enabling `[peer_private]` on your validator is that your validator's peers do not include it in their Peer Crawler results.
 
 
+## 5. Verify your network connection
+
+Here are some methods you can use to verify that your validator has a healthy connection to the XRP Ledger network:
+
+- Use the [`peers`](peers.html) command to return a list of all `rippled` servers currently connected to your validator. If the `peers` array is `null`, you don’t have a healthy connection to the network. If you've set up your validator using the instructions in this document, the `peers` array should include the same number of objects as the number of peers defined in your `[ips_fixed]` stanza.
+
+    If you listed a public hub in your `[ips_fixed]` stanza and it is busy, it may reject your validator's connection. In this case, you may end up with fewer connections than configured in your `[ips_fixed]` stanza. Your validator continues to connect to the public hub.
+
+    If you are having trouble maintaining a reliable and safe connection to the network and haven't set up connections using public hubs or proxies, see [4. Connect to the network](#4-connect-to-the-network]. Using one of the methods described in the section may help your validator remain healthily connected to the network.
```

Comment:
`[4. Connect to the network](#4-connect-to-the-network]`

Typo - last `]` should be `)` instead